### DISCUSSION TITLE
change anime model error

### DISCUSSION THREAD

**Discussion by killuams08** [lllyasviel/Fooocus#3049] [Q&A] [answered]
run by run.bat,if I change preset to anime,it will be break and show this error occurred
RuntimeError: [enforce fail at ..\c10\core\impl\alloc_cpu.cpp:72] data. DefaultCPUAllocator: not enough memory: you tried to allocate 3276800 bytes.

but I try run by run_anime.bat，This error did not occur，if I change the modle to bluepencile or other，It show again，this error never occurred in other version。
![Snipaste_2024-05-30_17-48-03](https://github.com/lllyasviel/Fooocus/assets/28971393/d2835901-087b-4f0a-aa75-6e78e44ddf3a)

**Comment by mashb1t** [comment]
@killuams08 can you please provide the startup terminal output of both the run.bat and run_anime.bat to check system resource allocation?

**Reply by killuams08** [reply, marked as answer]
> @killuams08 can you please provide the startup terminal output of both the run.bat and run_anime.bat to check system resource allocation?

It fix....I dont konw why,I just restart the computer....and It was fix .
